### PR TITLE
Post IA data in dax comment

### PR DESCRIPTION
**Pull request by jdorweiler**
Post some of the data from the IA page back to the PR in a dax comment. 
selection_104

This runs with ghIssues so data from the IA page would be synced every time it runs.  I have it set to only post to the first comment (excluding dax welcome message).  This means it won't post a comment if someone else comments before ghIssues runs.  So dax gets the first comment or doesn't post. 

@russellholt @MariagraziaAlastra

### CODE REVIEW

File: script/ghIssues.pl [hunk 1]
```diff
@@ -33,9 +33,9 @@ BEGIN
 
 # the repos we care about
 my @repos = (
-    'zeroclickinfo-spice',
-    'zeroclickinfo-goodies',
-    'zeroclickinfo-longtail',
+#    'zeroclickinfo-spice',
+#    'zeroclickinfo-goodies',
+#    'zeroclickinfo-longtail',
```
Comment (MariagraziaAlastra) [inline script/ghIssues.pl]: @jdorweiler Why are these repos commented out?

File: script/ghIssues.pl [hunk 2]
```diff
@@ -437,6 +435,56 @@ sub get_mentions {
...
 }
 
+sub update_pr_template {
+    my ($data, $pr_number, $source) = @_;
+
+    # find dax comment at spot #1 or bail
+    my @comments = $gh->issue->comments($pr_number);
+    my $comment_number;
+    if(scalar @comments){
+        my $comment = $comments[0];
```
Comment (MariagraziaAlastra) [inline script/ghIssues.pl]: @jdorweiler would this be the PR description?

**Comment by jdorweiler** [comment]
Updated template
selection_063

File: script/ghIssues.pl [hunk 3]
```diff
@@ -437,6 +435,100 @@ sub get_mentions {
...
+        if($val){
+            $val = '&#10003;';
+        }else{
+            $val = ' ';
+        }
+        $browsers .= '['.$val.'] ' . $browser. "\n";
+    }
+
+    my $other_tests;
+    foreach my $test (qw(code_review design_review)){
```
Comment (MariagraziaAlastra) [inline script/ghIssues.pl]: @jdorweiler Not sure this is actually helpful. Code review and design review don't necessarily exclude each other, and when a user submits a PR he won't have had any design or code review yet anyways.
Also we aren't using this info on IA Pages, nor we have specific plans about it for the near future.

**Comment by MariagraziaAlastra** [comment]
@jdorweiler This is some seriously great work, thank you! Let me know when it's ready for testing :cactus:

**Comment by jdorweiler** [comment]
Ready to go again.  See this line for testing though.
https://github.com/duckduckgo/community-platform/pull/1115/files#diff-0178963ea1787f16ec7f90819d463bc8R438

**Comment by jdorweiler** [comment]
We're going to need the dax token too.  I'll look into getting that added.

**Comment by jdorweiler** [comment]
selection_105

**Comment by jdorweiler** [comment]
Fixed check boxes
selection_106

**Comment by MariagraziaAlastra** [comment]
@jdorweiler Awesome work. I'm impressed by the detail level here, thanks for putting so much effort into this code. Tested, LGTM :+1:

**Pull request merged**